biome.jsonc

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "vcs": { "enabled": true, "clientKind": "git", "useIgnoreFile": true },
+  "files": {
+    "ignoreUnknown": true
+  },
+  "organizeImports": { "enabled": true },
+  "formatter": { "enabled": true, "useEditorconfig": true },
+  "css": { "formatter": { "quoteStyle": "single" } },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "a11y": {
+        "all": false
+      },
+      "suspicious": {
+        "noExplicitAny": "info",
+        "noGlobalIsNan": "warn",
+        "noImplicitAnyLet": "warn",
+        "noAssignInExpressions": "warn",
+        "noConfusingVoidType": "warn",
+        "useNamespaceKeyword": "warn" // safe fix
+      },
+      "style": {
+        "noUnusedTemplateLiteral": "warn",
+        "noInferrableTypes": "warn", // safe fix
```

Comment:
Did you apply these safe fixes ? If so no need to set them as warn, we want to avoid them as much as possible so an error is fine

Reply:
I've only applied the fixes that were causing formatting or import order errors, which I forgot to mention. Now, running `pnpm lint` exits with 0, while `pnpm lint:fix` would apply additional linting fixes. It might be better to address these in the next PR, as they are more nuanced and might not be necessary if some rules are disabled.